Makefile

```diff
@@ -274,148 +275,283 @@ docker-ci-integration:
 		-w "/go/flow" "$(CONTAINER_REGISTRY)/golang-cmake:v0.0.7" \
 		make ci-integration
 
-.PHONY: docker-build-collection
-docker-build-collection:
-	docker build -f cmd/Dockerfile  --build-arg TARGET=./cmd/collection --build-arg COMMIT=$(COMMIT)  --build-arg VERSION=$(IMAGE_TAG) --build-arg GOARCH=$(GOARCH) --build-arg CGO_FLAG=$(CRYPTO_FLAG) --target production \
+# only works on linux
+.PHONY: install-cross-build-tools
+install-cross-build-tools:
```

Comment:
this only works on debian hosts, which may be confusing since we also have a `install-tools` which works on mac. How about renaming to `install-cross-build-tools-debian` or something like that

Reply:
Ideally we would add an os check within the target, beyond the name. I'll try to add one

Reply:
https://github.com/onflow/flow-go/pull/5254/commits/0324c18f1fb162450cdf444471a40ad511dd7c58